### PR TITLE
Added Git Statue command

### PR DESCRIPTION
**Pull request by JoeyKanaly**
Used the last statue of liberty from https://www.asciiart.eu/buildings-and-places/monuments/statue-of-liberty

### CODE REVIEW

File: shellGitTypo.sh [hunk 1]
```diff
@@ -228,6 +259,9 @@ git() {
   elif [ "$1" = "stash" -a "$2" = "poop" ]; then
     echo "$gitTypoPoop"
     command git stash pop "${@:3}"
+  elif [ "$1" = "statue" ]; then
+    echo "$gitTypoStatue"
+    command git status
```
Comment (Xaxetrov) [inline shellGitTypo.sh]: Do not forget to also pass other arguments to the command.

File: shellGitTypo.sh [hunk 2]
```diff
@@ -228,6 +259,9 @@ git() {
   elif [ "$1" = "stash" -a "$2" = "poop" ]; then
     echo "$gitTypoPoop"
     command git stash pop "${@:3}"
+  elif [ "$1" = "statue" ]; then
```
Comment (Xaxetrov) [inline shellGitTypo.sh:266]: Please jump a line and also add a `# status` comment so that we can group all "status" typos together.

**Review by Xaxetrov — COMMENTED** [review]
Thanks for your contribution!

**Comment by JoeyKanaly** [comment]
I believe I fixed everything that you wanted, if not just let me know.

**Review by Xaxetrov — APPROVED** [review]
All good thanks ! Small typo in the first comment but I'll fix it later.

**Pull request merged**